modules/jetty-support/jetty-support.gradle

```diff
@@ -1,6 +1,7 @@
 // Adapted jetty related bundles
 platform {
 	def jettyVersion = '9.2.1.v20140609'
+	//def jettyVersion = '9.4.10.v20180503' // Originally upgraded during the platform migration to Photon. Not sure why anymore, so left out for the moment
```

Comment:
I observed a validaton error on the HALE product related to Jetty (something about requited capabilities). It did not seem related to my changes, so maybe it was a fix (or an attempt) for that problem.